src/shipper/index.ts

```diff
@@ -2,3 +2,15 @@ export { Log, version, hash } from '../logger';
 export { handler, S3 } from './app';
 export { ElasticSearch } from './elastic';
 export { LogShipper } from './shipper.config';
+export { LogObject } from './type';
+
+// Inject additional log functions if there some provided
+import { LogShipper } from './shipper.config';
+import { LogFunctionName } from './type';
+try {
+  const AdditionalLogFuncs = require(`./${LogFunctionName}`) ?? [];
+  console.log('FoundAdditionalFunctions', AdditionalLogFuncs);
+  for (const fn of AdditionalLogFuncs) LogShipper.DefaultLogProcessFunctions.push(fn);
+} catch (e) {
+  //ignore
+}
```

Comment:
Could add this
```typescript
if (e.code !== 'MODULE_NOT_FOUND') {
  throw e;
}
```

Reply:
Good idea

Reply:
Adding this, throws a `ENOENT` and just catching that throws a different error, might be easier to just throw